dbt/include/snowflake/macros/materializations/snapshot.sql

```diff
@@ -1,8 +1,9 @@
 {% materialization snapshot, adapter='snowflake' %}
     {% set original_query_tag = set_query_tag() %}
+    {% set  grant_config = config.get('grants') %}
 
     {% set relations = materialization_snapshot_default() %}
-
+    {% do apply_grants(target_relation, grant_config, should_revoke=True) %}
```

Comment:
Ditto - this will be handled within `materialization_snapshot_default`, so not needed here